CloudFormation/amazonmq.yml

```diff
@@ -453,16 +481,17 @@ Resources:
       Users:
         - Username: !Sub "{{resolve:secretsmanager:${MasterSecret}:SecretString:username}}"
           Password: !Sub "{{resolve:secretsmanager:${MasterSecret}:SecretString:password}}"
-          Groups: !Split [ ",", !Sub "{{resolve:secretsmanager:${MasterSecret}:SecretString:groups}}" ]
+          Groups: [ !Sub "{{resolve:secretsmanager:${MasterSecret}:SecretString:group}}" ]
```

Comment:
Is the `MasterSecret` supposed to be used by all services, or is it only for Alfresco?